tests/test_gordian.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, config_file='./tests/fixtures/test_config.yaml', dry_run = Fa
             self.github_api = None
             self.semver_label = None
             self.target_branch = 'master'
+            self.pr_labels = ['test']
```

Comment:
Can you add a test without this set to ensure that the default `[]` also works?

Reply:
Added in: https://github.com/argoproj-labs/gordian/pull/22/commits/2d1f36eb004bc1ba61026cfdd75e3d75bfb395fe 🍰